### PR TITLE
Assorted mappings

### PR DESCRIPTION
**Pull request by pyrox0**

### CODE REVIEW

File: mappings/net/minecraft/enchantment/Enchantment.mapping [hunk 1]
```diff
@@ -1,12 +1,32 @@
 CLASS net/minecraft/unmapped/C_jxtrubuh net/minecraft/enchantment/Enchantment
+	FIELD f_cvunyugc HOLDER_CODEC Lcom/mojang/serialization/Codec;
```
Comment (eristhea) [inline mappings/net/minecraft/enchantment/Enchantment.mapping:2]:
```suggestion
	FIELD f_cvunyugc ENTRY_CODEC Lcom/mojang/serialization/Codec;
```
this is the mapping I use in #598, as when you say holder it's not clear what exactly it's holding

File: mappings/net/minecraft/enchantment/Enchantment.mapping [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+	FIELD f_cytkilek properties Lnet/minecraft/unmapped/C_jxtrubuh$C_ryljcnwe;
```
Comment (eristhea) [inline mappings/net/minecraft/enchantment/Enchantment.mapping:3]:
```suggestion
	FIELD f_cytkilek createProperties Lnet/minecraft/unmapped/C_jxtrubuh$C_ryljcnwe;
```
convention

File: mappings/net/minecraft/enchantment/Enchantment.mapping [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+	FIELD f_yozrarfo CODEC Lcom/mojang/serialization/Codec;
+	METHOD m_ajxstdmb properties (Lnet/minecraft/unmapped/C_odfnijdo;Lnet/minecraft/unmapped/C_odfnijdo;IILnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;Lnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;I[Lnet/minecraft/unmapped/C_qfoqahef;)Lnet/minecraft/unmapped/C_jxtrubuh$C_ryljcnwe;
```
Comment (eristhea) [inline mappings/net/minecraft/enchantment/Enchantment.mapping:5]:
```suggestion
	METHOD m_ajxstdmb createProperties (Lnet/minecraft/unmapped/C_odfnijdo;Lnet/minecraft/unmapped/C_odfnijdo;IILnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;Lnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;I[Lnet/minecraft/unmapped/C_qfoqahef;)Lnet/minecraft/unmapped/C_jxtrubuh$C_ryljcnwe;
```
again

File: mappings/net/minecraft/enchantment/Enchantment.mapping [hunk 4]
```diff
@@ -1,12 +1,32 @@
...
 	METHOD m_khmhdqpw isAcceptableItem (Lnet/minecraft/unmapped/C_sddaxwyk;)Z
 		COMMENT {@return the type of item ({@code stack}) that can have this applied to it}
 		ARG 1 stack
 	METHOD m_lcxutjum getWeight ()I
+	METHOD m_nfuegjdm (Lnet/minecraft/unmapped/C_jxtrubuh$C_ryljcnwe;)Lnet/minecraft/unmapped/C_jxtrubuh$C_epmoqirc;
+		ARG 0 properties
+	METHOD m_npijgbpy getSupportedItems ()Lnet/minecraft/unmapped/C_odfnijdo;
+	METHOD m_qirqbtfb (Ljava/util/List;Lnet/minecraft/unmapped/C_iakykpgh;Ljava/util/function/Consumer;)V
+		ARG 1 lootContext
+	METHOD m_rftiggzk properties (Lnet/minecraft/unmapped/C_odfnijdo;IILnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;Lnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;I[Lnet/minecraft/unmapped/C_qfoqahef;)Lnet/minecraft/unmapped/C_jxtrubuh$C_ryljcnwe;
```
Comment (eristhea) [inline mappings/net/minecraft/enchantment/Enchantment.mapping:29]:
```suggestion
	METHOD m_rftiggzk createProperties (Lnet/minecraft/unmapped/C_odfnijdo;IILnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;Lnet/minecraft/unmapped/C_jxtrubuh$C_jkttgiwc;I[Lnet/minecraft/unmapped/C_qfoqahef;)Lnet/minecraft/unmapped/C_jxtrubuh$C_ryljcnwe;
```
one more

File: mappings/net/minecraft/loot/condition/EnchantmentActiveCheckLootCondition.mapping [hunk 5]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_myfyjuce net/minecraft/loot/condition/EnchantmentActiveCheckLootCondition
```
Comment (eristhea) [inline mappings/net/minecraft/loot/condition/EnchantmentActiveCheckLootCondition.mapping:1]: maybe `CheckEnchantmentActive`? that's more natural

File: mappings/net/minecraft/loot/function/EnchantedCountIncreaseLootFunction.mapping [hunk 6]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_edcogzgr net/minecraft/loot/function/EnchantedCountIncreaseLootFunction
```
Comment (eristhea) [inline mappings/net/minecraft/loot/function/EnchantedCountIncreaseLootFunction.mapping:1]: same here, this is difficult to parse

File: mappings/net/minecraft/loot/condition/RandomChanceWithEnchantedBonusLootCondition.mapping [hunk 7]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_cbexmsqd net/minecraft/loot/condition/RandomChanceWithEnchantedBonusLootCondition
```
Comment (eristhea) [inline mappings/net/minecraft/loot/condition/RandomChanceWithEnchantedBonusLootCondition.mapping:1]: this is hard to understand, despite the long name
what does it do?

File: mappings/net/minecraft/network/listener/PacketListener.mapping [hunk 8]
```diff
@@ -5,8 +5,8 @@ CLASS net/minecraft/unmapped/C_tkgdawqw net/minecraft/network/listener/PacketLis
 	COMMENT <p>Its listener methods will be called on the netty event loop than the
 	COMMENT client or server game engine threads.
 	METHOD m_eqgvuckk getState ()Lnet/minecraft/unmapped/C_kxdobmrm;
-	METHOD m_ksdgvxqp addCrashReportInfo (Lnet/minecraft/unmapped/C_bkefkbum;Lnet/minecraft/unmapped/C_qympisds;)V
```
Comment (eristhea) [inline mappings/net/minecraft/network/listener/PacketListener.mapping:8]: why is this removed?

**Review by eristhea — COMMENTED** [review]
nice to see more mappings from you :)

File: mappings/net/minecraft/network/listener/PacketListener.mapping (hunk 8 above)
Comment (pyrox0) [reply mappings/net/minecraft/network/listener/PacketListener.mapping:8]: Not sure why it was, reverted.

File: mappings/net/minecraft/block/Blocks.mapping [hunk 9]
```diff
@@ -32,10 +34,16 @@ CLASS net/minecraft/unmapped/C_jricjyva net/minecraft/block/Blocks
 		ARG 0 sticky
 	METHOD m_sidkwvli createStoneButton ()Lnet/minecraft/unmapped/C_mmxmpdoq;
 	METHOD m_tetaodtf rebuildShapeCache ()V
+	METHOD m_tldmwsps (Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_peaveboq;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_ogavsvbr;)Z
+		ARG 3 entityType
+	METHOD m_tobwsgju mapColorOf (Lnet/minecraft/unmapped/C_mlrprqox;)Ljava/util/function/Function;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/Blocks.mapping:39]: I think "create" is more conventional when not creating an instance of the class the method is in. #599 also mentions "waterlogged" for this.

**Review by supersaiyansubtlety — COMMENTED** [review]
Just one note, and conflicts with #599

**Comment by pyrox0** [comment]
Closing per what I said in the mappings thread. Feel free to take these changes and use em yourself.
